audit_exceptions/linux_only_gcc_dependency_allowlist.json

```diff
@@ -1,4 +1,5 @@
 [
   "nghttp2",
+  "libunicode",
   "poac"
```

Comment:
Just another note here: the reason why adding `nghttp2` and `poac` here works fine is that they just ship executables, so they won't force dependents to use a newer `libstdc++`.